### ISSUE TITLE
Populate Engineer sections of Alt Text Rollout Strategy Page

### ISSUE THREAD

**Issue by maortiz-27-80** [department-of-veterans-affairs/va.gov-cms#16583]
## User Story or Problem Statement
As a person who works on Va.gov, I should be able to go to a single source of truth (page) to understand the impact and expectations of the new alt text validation feature. 

## Description or Additional Context 
Fill out the github page for the alt text rollout strategy: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/rollout-strategy/alt-text-validation.md

## Acceptance Criteria
- [x] Engineer tasks H2 - checklist of tasks
- [x] Prelaunch H2 - any relevant information, re: code merge freeze, PR reviews
- [x] Day of Launch H2 - Tasks, Start Time, End Time, and duration
- [x] Post-Launch H2 - checklist of tasks

## Team
Please check the team(s) that will do this work.

- [X] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

cc: @BerniXiongA6 @ndouglas

**Comment by maortiz-27-80** [comment]
Marking as complete and closing. cc: @BerniXiongA6 @edmund-dunn
